tests/tensor_search/integ_tests/test_search_unstructured.py

```diff
@@ -1375,3 +1375,42 @@ def test_lexical_query_can_not_be_none(self):
                 with self.assertRaises(InvalidArgError):
                     res = tensor_search.search(text=None, config=self.config, index_name=self.default_text_index,
                                                search_method=SearchMethod.LEXICAL)
+                                                    
+    def test_empty_lexical_query(self):
```

Comment:
1. Move these tests to `test_search_combined`. We're trying to move away from files specifically for structured/unstructured
2. separate the tests for `*` from the ones for `""`, with descriptive test names
3. Make test for wildcard with filter
4. Make test for the edge cases mentioned above (wildcard `or term` with text `and term`, vice verse)

Reply:
created!